vagrant/provisioning/arkcase-all.yml

```diff
@@ -70,6 +70,39 @@
       tags: [arkcase-prerequisites, core, arkcase, upgrade, marketplace]
     - role: arkcase-app
       tags: [arkcase-app, core, arkcase, upgrade, marketplace]
+    - role: arkcase-api-gateway
+      tags: [core, upgrade, arkcase-api-gateway, arkcase-microservices, marketplace]
```

Comment:
Is "yes" preferred over a boolean type?  e.g. `when: enable_kafka is defined and enable_kafka|bool`?

Reply:
Also, maybe `enable_microservices` versus `enable_kafka`?

Reply:
We have enable_kafka variable, so I reused it here. My thinking was that without Kafka you cannot have microservices.
The existing variable was "yes" and didn't wan't to change it. I prefer boolean true/false. Having said this, I can change this, because it won't really change much at this point. So, I'll introduce enable_microservices boolean variable and get rid of enable_kafka variable.